libs/eo/help/shell/src/lib/eo-geography-page.component.ts

```diff
@@ -15,12 +15,192 @@
+              dokumentation og sporing af energis oprindelse, for at sikre at
+              energien er grøn, er det vigtigt, at der tages højde for disse
+              udfordringer.
+            </p>
+          </div>
+          <div>
+            <h4>Grøn produktion kan være sort forbrug</h4>
+            <p>
+              En af problematikkerne findes bl.a. ved køb af GO certifikater.
+              Det er f.eks. lige nu muligt for danske virksomheder at købe GO
+              certifikater af grøn energi produceret i Island. På papiret ser
+              det på den måde ud som om, at den energi man forbruger er grøn.
+              Virkeligheden kan dog være en helt anden. Da der ikke findes nogen
+              forbindelse mellem Island og Danmark, så energien produceret i
+              Island kan umuligt være samme energi forbrugt i Danmark.
+            </p>
+          </div>
+          <div class="img-box">
+            <img
+              width="367"
+              src="/assets/images/help/map_geography.jpg"
+              alt="Transmission net across countries"
+            />
+            <span>
+              Billedet viser det europæiske transmissionsnet på tværs af
+              landegrænser
+            </span>
+          </div>
+        </eo-stack>
+      </div>
+      <div class="text-box">
+        <span class="watt-headline-3">Hvordan løser vi udfordringerne?</span>
+        <p>
+          En løsning er at sætte regler for, hvor lang afstanden må være fra
+          produktions- til forbrugsstedet ved handel af grønne certifikater.
+          Begrænsningerne sættes af de fysiske kabler, altså at man kun kan
+          handle certifikater fra nabolande. På den måde sikrer man, at grøn
+          energi kun kan bruges i produktionslandet og eventuelle nabolande.
+        </p>
+      </div>
+      <div class="case">
+        <img
+          alt="Geography case"
+          src="/assets/images/help/case_geography.jpg"
+        />
+        <p>
+          <b>CASE</b><br />
+          Peter Producent kan sælge sine certifikater i Danmark, Tyskland, Norge
+          og Sverige. Så længe der er plads på de respektive forbindelser.
+        </p>
+      </div>
+      <h4>
+        <a class="link" routerLink="../{{ routes.introduction }}">
+          << Tilbage til Introduktion til EnergiOprindelse
+        </a>
+      </h4>
+    </div>
+    <div class="column">
+      <div>
+        <eo-stack size="M">
+          <p
+            class="watt-headline-1"
+            style="color: var(--watt-color-primary-light);"
+          >
+            Geografi
+          </p>
+          <p style="color: var(--watt-color-primary-dark)">
+            Der er enkelte fysiske begrænsninger for, hvordan energi kan bevæge
+            sig rundt, som bliver nødt til at være indarbejdet i systemet.
+          </p>
+        </eo-stack>
+      </div>
+
+      <div class="info-box green">
```

Comment:
Sounds like a good component with some display settings.
Would avoid specifying the colors, but maybe use "low contrast, high contrast etc, to allow a different color scheme"

Reply:
Made a new box :)